tests/python/accuracy/prepare_data.py

```diff
@@ -1,76 +1,103 @@
+                client,
+                f"https://storage.openvinotoolkit.org/repositories/model_api/test/otx_models/{model_name}/openvino.bin",
+                f"{otx_models_dir}/{model_name}.bin",
+            ),
+        )
 
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Data and model prepare script")
+async def main():
+    parser = argparse.ArgumentParser()
     parser.add_argument(
         "-d",
-        dest="data_dir",
-        default="./data",
+        "--data_dir",
+        type=Path,
+        required=True,
         help="Directory to store downloaded models and datasets",
     )
-
     args = parser.parse_args()
 
-    prepare_data(args.data_dir)
-    retrieve_otx_model(args.data_dir, "mlc_mobilenetv3_large_voc")
-    retrieve_otx_model(args.data_dir, "mlc_efficient_b0_voc")
-    retrieve_otx_model(args.data_dir, "mlc_efficient_v2s_voc")
-    retrieve_otx_model(args.data_dir, "det_mobilenetv2_atss_bccd")
-    retrieve_otx_model(args.data_dir, "cls_mobilenetv3_large_cars")
-    retrieve_otx_model(args.data_dir, "cls_mobilenetv3_large_cars", "onnx")
-    retrieve_otx_model(args.data_dir, "cls_efficient_b0_cars")
-    retrieve_otx_model(args.data_dir, "cls_efficient_v2s_cars")
-    retrieve_otx_model(args.data_dir, "Lite-hrnet-18")
-    retrieve_otx_model(args.data_dir, "Lite-hrnet-18_mod2")
-    retrieve_otx_model(args.data_dir, "Lite-hrnet-s_mod2")
-    retrieve_otx_model(args.data_dir, "Lite-hrnet-x-mod3")
-    retrieve_otx_model(args.data_dir, "is_efficientnetb2b_maskrcnn_coco_reduced")
-    retrieve_otx_model(args.data_dir, "is_resnet50_maskrcnn_coco_reduced")
-    retrieve_otx_model(args.data_dir, "mobilenet_v3_large_hc_cf")
-    retrieve_otx_model(args.data_dir, "classification_model_with_xai_head")
-    retrieve_otx_model(args.data_dir, "detection_model_with_xai_head")
-    retrieve_otx_model(args.data_dir, "segmentation_model_with_xai_head")
-    retrieve_otx_model(args.data_dir, "maskrcnn_model_with_xai_head")
-    retrieve_otx_model(args.data_dir, "maskrcnn_xai_tiling")
-    retrieve_otx_model(args.data_dir, "tile_classifier")
-    retrieve_otx_model(args.data_dir, "anomaly_padim_bottle_mvtec")
-    retrieve_otx_model(args.data_dir, "anomaly_stfpm_bottle_mvtec")
-    retrieve_otx_model(args.data_dir, "deit-tiny")
+    otx_models_dir = args.data_dir / "otx_models"
+    otx_models_dir.mkdir(parents=True, exist_ok=True)
+    async with httpx.AsyncClient(timeout=20.0) as client:
+        await asyncio.gather(
+            download_images(args.data_dir),
+            download_otx_model(client, otx_models_dir, "mlc_mobilenetv3_large_voc"),
+            download_otx_model(client, otx_models_dir, "mlc_efficient_b0_voc"),
+            download_otx_model(client, otx_models_dir, "mlc_efficient_v2s_voc"),
+            download_otx_model(client, otx_models_dir, "det_mobilenetv2_atss_bccd"),
+            download_otx_model(client, otx_models_dir, "cls_mobilenetv3_large_cars"),
```

Comment:
Where is onnx version of "cls_mobilenetv3_large_cars"?

Reply:
Added